.editorconfig

```diff
@@ -8,3 +8,6 @@ insert_final_newline = true
 charset = utf-8
 indent_style = space
 indent_size = 2
+
+[*.bat]
+end_of_line = crlf
```

Comment:
I'm suprised why we have `.editorconfig` in project. It is not compatible with Developer box.